packages/x-components/src/x-modules/search/store/actions/set-params-from-url.action.ts

```diff
@@ -24,4 +26,8 @@ export const setParamsFromUrl: SearchXStoreModule['actions']['setParamsFromUrl']
   if (newPage) {
     commit('setPage', newPage);
   }
+
+  if (newFilters) {
+    commit('setSelectedFilters', createRawFilters(newFilters));
```

Comment:
Is this necessary? Doesn't the facet's module communicate the selected filters in time for the first request?

Reply:
And if not, we have to fix it in another way. 

We can not have the facets module and the search module doing the same.